src/main/java/uk/gov/companieshouse/filinghistory/api/mapper/upsert/AbstractTransactionMapperFactory.java

```diff
@@ -29,10 +34,7 @@ public AbstractTransactionMapper getTransactionMapper(TransactionKindEnum kind)
             case TOP_LEVEL -> topLevelTransactionMapper;
             case ANNOTATION -> annotationTransactionMapper;
             case ASSOCIATED_FILING -> associatedFilingTransactionMapper;
-            case RESOLUTION -> {
-                LOGGER.error("Invalid transaction kind: %s".formatted(kind.getValue()), DataMapHolder.getLogMap());
-                throw new InvalidTransactionKindException("Invalid transaction kind: %s".formatted(kind.getValue()));
-            }
+            case RESOLUTION -> resolutionTransactionMapper;
```

Comment:
If the enum gets added to we will need a default case to cover it. It should probably do what the previous resolution case was doing, logging and throwing exception.

Reply:
Intellij doesn't like having a default branch there but done.

Reply:
resolved in call.